lib/pacio_test_kit/common_tests/adi_search_tests/patient_search_tests/patient_family_gender_search_test.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../../searcht_test'
+
+module PacioTestKit
+  class PatientFamilyGenderSearchTest < Inferno::Test
+    include PacioTestKit::SearchTest
+
+    title 'Server returns valid results for search by family + gender'
+    description %(
+      A server SHALL support searching by family + gender.
```

Comment:
The capability statement references [US Core Patient V7.0.0](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html). In this US Core version, `family + gender` has a `SHOULD` conformance. Fix the title and mark this test as optional.